source/gui/settingsDialogs.py

```diff
@@ -1056,75 +1057,109 @@ class DocumentFormattingDialog(SettingsDialog):
 	title = _("Document Formatting")
 
 	def makeSettings(self, settingsSizer):
+
+		sHelper = guiHelper.BoxSizerHelper(self, sizer=settingsSizer)
+
+		# Translators: This is a label appearing on the document formatting settings dialog.
+		dialogText =_("The following options control the types of document formatting reported by NVDA.")
+		sHelper.addItem(wx.StaticText(self, label=dialogText))
+
+		scrolledItems = scrolledpanel.ScrolledPanel(self, size = (500,400), style = wx.TAB_TRAVERSAL | wx.BORDER_THEME)
+		scrolledItems.SetAutoLayout(True)
+		scrolledItems.SetupScrolling()
+		self.scrolledItems = scrolledItems
+		checkboxItems = guiHelper.BoxSizerHelper(scrolledItems, orientation=wx.VERTICAL)
+		checkboxItemsBorder = wx.BoxSizer(wx.VERTICAL)

```

Comment:
`checkboxItems` does mostly contain check boxes, but the line indentation setting is a choice control, not a check box. Therefore, I think this should be renamed, as I'm concerned it might be misleading in future. Maybe we could call this `scrolledItems` and rename the panel to `scrolledPanel`?